examples/react-router/app/boot.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react'
 import DOM from 'react-dom'
-import Router from 'react-router-dom/BrowserRouter'
-import Debugger from 'microcosm-debugger'
+import { Router } from 'react-router'
+import createBrowserHistory from 'history/createBrowserHistory'
 import { AppContainer } from 'react-hot-loader'
 import Repo from './repo'
 import Layout from './views/layout'
 
 const el = document.getElementById('app')
-const repo = new Repo({ maxHistory: Infinity })
-
-Debugger(repo)
+const browserHistory = createBrowserHistory()
+const repo = new Repo({ maxHistory: Infinity, browserHistory })
```

Comment:
By sending in the same history to our Repo and React Router, the Routing effect can manage navigation. This is also very easy to test:

```javascript
import { createMemoryHistory } from 'history'

let repo = new Repo({ browserHistory: createMemoryHistory() })

// .. do other things, and we don't need a real browser. We can
// also assert on history
```